portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
It would have been better to put the servlet file in a different PR.

Reply:
Thank you! It didn't occur to me to do this at the time. This would absolutely help, because then my html can be incorporated before "moving on"... (I decided to work ahead but can definitely see why that would have been advantageous... I will do that in the future!)